src/providers/data_provider_be.c

```diff
@@ -549,6 +549,14 @@ errno_t be_process_init(TALLOC_CTX *mem_ctx,
         goto done;
     }
 
+    /* Load the resolv.conf file in case a call to dbus' resInit() was missed */
+    if (monitor_be_methods.resInit != NULL) {
+        ret = monitor_be_methods.resInit(NULL, (void *) be_ctx);
```

Comment:
Why not simply `data_provider_res_init()`?

Reply:
Well, maybe this is indeed more consistent.

Reply:
That's what I think.